packages/webdriverio/src/commands/element/isDisplayed.js

```diff
@@ -41,5 +41,5 @@
  */
 
 export default function isDisplayed() {
-    return this.isElementDisplayed(this.elementId)
+    return this.elementId ? this.isElementDisplayed(this.elementId) : false
```

Comment:
if called on "No Such Element" it will immediately return false.

Reply:
This information could be outdated. I think it would make more sense to refetch the element at least once to see if it appeared in the time between it was fetched and the command called.

Reply:
Is this for people who cache the element prior to referencing it? In a lazyloading scenario it should be within milliseconds (depending on network latency) between the fetch and the check.

Reply:
done